src/busio.py

```diff
@@ -16,7 +16,7 @@ def __init__(self, scl, sda, frequency=400000):
 
     def init(self, scl, sda, frequency):
         self.deinit()
-        if board_id == "raspi_3" or board_id == "raspi_2" or board_id == "beaglebone_black":
+        if board_id == "raspi_3" or board_id == "raspi_2" or board_id == "beaglebone_black" or board_id == "orangepipc":
```

Comment:
might be more pythonic here to say `if board_id in ("raspi_3", "raspi_2", "beaglebone_black", "orangepipc")`.